### ISSUE TITLE
Can't print text on title screen

### ISSUE THREAD

**Issue by warrigal24** [ainslec/adventuron-issue-tracker#260]
All games start with a title screen. This is normally a custom image. If a custom image is not provided, then the Adventuron logo is shown. This is all very fine. but there are no instructions to tell you to click the mouse button to continue. You should be able to provide text below the image to tell the player how to get past that screen.

**Comment by ainslec** [comment]
I agree it's a minor issue but it's not a minor fix so will be reviewed when the renderer is revisited.
